lighthouse-cli/cli-flags.ts

```diff
@@ -77,6 +77,7 @@ export function getFlags(manualArgv?: string) {
             CHROME_PATH: Explicit path of intended Chrome binary. If set must point to an executable of a build of Chromium version 54.0 or later. By default, any detected Chrome Canary or Chrome (stable) will be launched.
             `,
         'perf': 'Use a performance-test-only configuration',
+        'host': 'The host to use for the debugging protocol.',
```

Comment:
should copy this to https://github.com/GoogleChrome/lighthouse/blob/master/readme.md#cli-options, too (like #2757 just did for `CHROME_PATH`)